dev/add-to-output-repos.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+'use strict';
+
+const co = require('co');
+const _exec = require('child_process').exec;
+const fs = require('fs-extra');
+
+const stableBranch = 'stable';
+const betaBranch = 'master';
+
+function exec(command) {
+  return new Promise((resolve, reject) => {
+    _exec(command, (err, stdout) => {
+      if (err) {
+        return reject(err);
+      }
+
+      resolve(stdout.trim());
+    });
+  });
+}
```

Comment:
any reasons for not using `execa` instead?

Reply:
It wasn't working for the git clone for some reason.